test/rules/noAnyRuleTests.ts

```diff
@@ -15,14 +15,39 @@
  */
 
 describe("<no-any>", () => {
+    const fileName = "rules/noany.test.ts";
     const NoAnyRule = Lint.Test.getRule("no-any");
-    const fileName = "rules/noAny.test.ts";
+    const actualFailures = Lint.Test.applyRuleOnFile(fileName, NoAnyRule);
 
-    it("const declaration with type of 'any' should not be allowed", () => {
-        const actualFailures = Lint.Test.applyRuleOnFile(fileName, NoAnyRule);
-        const expectedFailure = Lint.Test.createFailure(fileName, [1, 8], [1, 11], "type decoration of 'any' is forbidden");
+    const createFailure = Lint.Test.createFailuresOnFile(fileName, NoAnyRule.FAILURE_STRING);
 
+    it("disallows variables with type 'any'", () => {
+        let expectedFailures = [createFailure([1, 8], [1, 11]),
```

Comment:
`const`

also I prefer having the brackets on their own line, it's easier to rearrange and delete elements of the array